api/src/main/resources/workbench.yaml

```diff
@@ -2159,10 +2175,58 @@ paths:
           schema:
             $ref: "#/definitions/DataSetPreviewResponse"
 
+  /v1/offline/syncResearcherData:
```

Comment:
[Non-opt] Naming nitpick: For consistency let's align on a single verb. Prefer using "export" rather than "sync" or "send", e.g. /v1/rdrExport/exportResearcherData, or even /v1/rdrExport/exportResearchers which makes it obvious that the input is a list.

This applies equally to the workspaces method name / URL path.